python/requirements/test-requirements.txt

```diff
@@ -18,7 +18,7 @@ cloudpickle==2.2.0
 # Keep in sync with `ci/build/upload_build_info.sh`
 cryptography==38.0.1
 cython==0.29.32
-fastapi[all]==0.99.1
+fastapi==0.104.0
```

Comment:
what is the difference with "[all]"?

Reply:
`[all]` pulls in some more optional dependencies that appear to be unnecessary

I removed it because it was causing me more headache trying to get the pip dependencies to compile